compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/GPULoweringConfigUtils.h

```diff
@@ -28,6 +28,15 @@ void setSubgroupNCount(MLIRContext *context,
                        SmallVectorImpl<NamedAttribute> &attrs,
                        int64_t subgroupNCount);
 
+// Helper to retrieve/set distribution basis.
+LogicalResult getBasis(IREE::GPU::LoweringConfigAttr config,
+                       IREE::GPU::TilingLevel level,
+                       SmallVector<int64_t> &basis,
+                       SmallVector<int64_t> &mapping);
+void setBasis(MLIRContext *context, SmallVector<NamedAttribute> &attrs,
+              IREE::GPU::TilingLevel level, ArrayRef<int64_t> basis,
+              ArrayRef<int64_t> mapping);
```

Comment:
nit: `mapping` is not very descriptive -- what is being mapped to what? Could you give it a different name or add a comment explaining it?

Reply:
added a comment explaining what it is